framework/views/errorHandler/callStackItem.php

```diff
@@ -10,35 +10,49 @@
 /* @var $args array */
 /* @var $handler \yii\web\ErrorHandler */
 ?>
-<li class="<?php if ($index === 1 || !$handler->isCoreFile($file)) echo 'application'; ?> call-stack-item"
+<li class="<?php if ($index === 1 || !$handler->isCoreFile($file)) {
+    echo 'application';
+} ?> call-stack-item"
     data-line="<?= (int) ($line - $begin) ?>">
     <div class="element-wrap">
         <div class="element">
             <span class="item-number"><?= (int) $index ?>.</span>
-            <span class="text"><?php if ($file !== null) echo 'in ' . $handler->htmlEncode($file); ?></span>
+            <span class="text"><?php if ($file !== null) {
+                echo 'in ' . $handler->htmlEncode($file);
+} ?></span>
             <span class="at">
-                <?php if ($line !== null) echo 'at line'; ?>
```

Comment:
This is what `PHPCBF` thinks it should look like. Haven't reviewed this myself yet.